app.rb

```diff
@@ -71,11 +60,11 @@ def create_student
 
   def create_teacher
     puts 'Create a new teacher'
-    print 'Enter teacher age'
+    print 'Enter teacher age: '
     age = gets.chomp.to_i
-    print 'Enter teacher name'
+    print 'Enter teacher name: '
     name = gets.chomp
-    print 'Enter teacher specialization'
+    print 'Enter teacher specialization: '
```

Comment:
- [ ] also in the `create_teacher` and `create_student` methods you are printing to the console, getting user data and processing the data, this is not in line with the single responsibility rule please have a look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more details about this rule.

Reply:
We are not getting `printing to the console, getting user data, and processing the data, this is not in line with the single responsibility rule`. Can you please explain it? From the articles, we couldn't get it.

Reply:
@mikethreels, We are not getting `printing to the console, getting user data, and processing the data, this is not in line with the single responsibility rule`. Can you please explain it? From the articles, we couldn't get it.

Reply:
@mikethreels, Thank you for your explanation. We get your point. We tried our best to follow single responsibility in the latest commit.